website/src/commands/append.md

```diff
@@ -36,6 +38,11 @@ main
 *   feature-2
 ```
 
+### --prototype / -p
```

Comment:
What's the motivation to move this block up? Right now they are sorted alphabetically. I'm fine with a different sorting order, as long as its clear which one it is so that I can follow it too :slightly_smiling_face:

Reply:
The motivation is to move arguments common to many commands to the end of the list. Specifically, I moved any instances of `[-d | --detached]`, `[--dry-run]`, and `[-v | --verbose]` to the end of the options list.

I think users visiting a page will be more interested in the options specific to that command.

I added a style guide to DEVELOPMENT.md so we can both remember the sorting decision: 728ecb379

Reply:
I like it, thanks for improving this!